.travis.yml

```diff
@@ -57,5 +57,6 @@ addons:
     postgresql: "9.3"
 
 script:
+    - make clean_travis
     - if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then make travis; fi
     - make test
```

Comment:
I'm not sure what you mean here by the scripts/commands exit unexpectedly, but we can make some cleanup after the tests too.